src/shortcuts/__tests__/undo.ts

```diff
@@ -69,6 +74,45 @@ it('undo thought change', () => {
   expect(exported).toEqual(expectedOutput)
 })
 
+it('persists undo thought change', async () => {
+
+  initialize()
+
+  fakeTimer.useFakeTimer()
+
+  appStore.dispatch([
+    importText({
+      path: RANKED_ROOT,
+      text: `
+        - a
+        - b`
+    }),
+    setCursorFirstMatchActionCreator(['a']),
+  ]
+  )
+  await fakeTimer.runAllAsync()
```

Comment:
Maybe this can be removed and the actions can be combined into one dispatch once `initialize` is awaited?